Website/DesktopModules/Hotcakes/Core/Admin/Parts/PaymentMethods/PaypalExpress/Edit.ascx

```diff
@@ -44,13 +39,9 @@
 <div class="hcColumnRight" style="width: 50%;">
     <div class="hcForm">
         <div class="hcFormItem">
-            <asp:Label runat="server" resourcekey="APISignature" CssClass="hcLabel" />
-            <asp:TextBox ID="txtSignature" runat="server" />
-            <asp:RequiredFieldValidator ID="rfvSignature" runat="server" resourcekey="rfvSignature" ControlToValidate="txtSignature" Display="Dynamic" CssClass="hcFormError" />
-        </div>
-        <div class="hcFormItem">
-            <asp:Label runat="server" resourcekey="PayPalFastSignupEmail" CssClass="hcLabel" />
-            <asp:TextBox ID="txtPayPalFastSignupEmail" runat="server" />
+            <asp:Label runat="server" resourcekey="APISecret" CssClass="hcLabel" />
+            <asp:TextBox ID="txtSecret" runat="server" />
+            <asp:RequiredFieldValidator ID="rfvSecret" runat="server" resourcekey="rfvSecret" ControlToValidate="txtSecret" Display="Dynamic" CssClass="hcFormError" />
```

Comment:
I don't see a matching key/value pair for the error message.

Reply:
Please let me know if it is well know.